tutorial.css

```diff
@@ -80,9 +80,9 @@ in the center, so it's placed to the left. These are used with .knit_border and
 
 #exercise {
   display: block;
-  width: 98%;
+  width: 100%;
```

Comment:
Hmm, not sure why that would be. I have notice odd stuff occasionally, but there should be JavaScript to scale the iframes automatically.